### ISSUE TITLE
Wrong Expo Router path for the +not-found.tsx page

### ISSUE THREAD

**Issue by Savinvadim1312** [roninoss/create-expo-stack#342]
I create a new project with Expo Router and tabs template. In the root folder of the project, there is:
- app - the correct folder for expo router
- stack
  - app
    - +not-found.tsx
    
![image](https://github.com/danstepanov/create-expo-stack/assets/16936043/18cb0358-6588-445e-bfff-ef6c1461d02c)

The `stack` folder, as a sibling of the `app` folder seems wrong. Am I missing something?

### CES config
```json
{
  "cesVersion": "2.11.1",
  "projectName": "InstagramTest",
  "packages": [
    {
      "name": "expo-router",
      "type": "navigation",
      "options": {
        "type": "tabs"
      }
    },
    {
      "name": "nativewind",
      "type": "styling"
    },
    {
      "name": "supabase",
      "type": "authentication"
    }
  ],
  "flags": {
    "noGit": false,
    "noInstall": false,
    "overwrite": false,
    "importAlias": true,
    "packageManager": "npm",
    "eas": false
  },
  "packageManager": {
    "type": "npm",
    "version": "10.8.1"
  },
  "os": {
    "type": "Darwin",
    "platform": "darwin",
    "arch": "arm64",
    "kernelVersion": "24.0.0"
  }
}

```

**Comment by danstepanov** [comment]
Will take a look in a few

**Comment by danstepanov** [comment]
The `stack` folder should be removed during project generation. Not sure why it isn't. Will fix today.

**Comment by dannyhw** [comment]
Yeah will take a look when i get a moment

**Comment by dannyhw** [comment]
for future reference heres the command im using to test this 
`npx create-expo-stack myTestProject --expo-router --tabs --nativewind --supabase`

**Comment by danstepanov** [comment]
this has been fixed as of v2.11.7
